### PR TITLE
Implement Clock exercise for Crystal

### PR DESCRIPTION
**Pull request by marksiemers**

### CODE REVIEW

**Comment by marksiemers** [comment]
@ErikSchierboom - I'm trying to use configlet to get things set properly.

I keep getting an error when trying to sync:
```
bin/configlet sync -e clock
Cloning https://github.com/exercism/problem-specifications/... failure
Cloning into '/Users/mark/Library/Caches/exercism/configlet/problem-specifications'...
ERROR: Repository not found.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

I am able to clone the problem-specifications repo the old fashioned way:
```
git clone https://github.com/exercism/problem-specifications.git
Cloning into 'problem-specifications'...
remote: Enumerating objects: 12172, done.
remote: Counting objects: 100% (1227/1227), done.
remote: Compressing objects: 100% (376/376), done.
remote: Total 12172 (delta 989), reused 925 (delta 849), pack-reused 10945
Receiving objects: 100% (12172/12172), 2.68 MiB | 8.90 MiB/s, done.
Resolving deltas: 100% (7367/7367), done.
```

**Comment by ErikSchierboom** [comment]
The PR looks good. Let's figure out the `configlet` issue and then we can merge.

File: exercises/practice/clock/.meta/src/example.cr [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    "%02i:%02i" % [hour, minute]
+  end
+
+  def hour
+    (minutes / MINUTES_PER_HOUR).modulo(HOURS_PER_DAY)
+  end
+
+  def minute
+    minutes.modulo(MINUTES_PER_HOUR)
+  end
```
Comment (hiljusti) [inline exercises/practice/clock/.meta/src/example.cr:39]: TIL Crystal wraps negative numbers to positive during `modulo`

(Many other languages preserve sign)

**Review by hiljusti — APPROVED** [review]

**Comment by hiljusti** [comment]
Approving, although you should incorporate the changes @ee7 is suggesting.

I'm ok if this happens outside this PR, maybe throw it in an issue if you need to troubleshoot configlet sync.

**Comment by marksiemers** [comment]
I think the issue with configlet and cloning is the trailing slash. I tried cloning with and without it, and the trailing slash makes it fail:
```
➜  code git clone https://github.com/exercism/problem-specifications
Cloning into 'problem-specifications'...
remote: Enumerating objects: 12177, done.
remote: Counting objects: 100% (1125/1125), done.
remote: Compressing objects: 100% (354/354), done.
remote: Total 12177 (delta 911), reused 843 (delta 769), pack-reused 11052
Receiving objects: 100% (12177/12177), 2.69 MiB | 3.23 MiB/s, done.
Resolving deltas: 100% (7363/7363), done.
➜  code rm -rf problem-specifications
➜  code git clone https://github.com/exercism/problem-specifications/
Cloning into 'problem-specifications'...
ERROR: Repository not found.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

**Review by hiljusti — APPROVED** [review]

**Pull request merged**